metrics.go

```diff
@@ -0,0 +1,20 @@
+package exporter
+
+import "context"
+
+// PartitionInfo holds information about a partition in Kafka.
+type PartitionInfo struct {
+	Topic           string
+	PartitionID     string
+	CurrentOffset   int64
+	Lag             int64
+	ClientID        string
+	ConsumerAddress string
+}
+
+// PartitionInfoClient queries consumer groups and consumer group partitions
+// for their stats.
+type PartitionInfoClient interface {
```

Comment:
For namings would it be more intuitive to fix names like below? WDYT?

- `PartitionInfoClient` => `ConsumerGroupInfoClient`
- `ConsumerGroupClient` => `ConsumerGroupsCommandClient`
- `FanInPartitionInfoClient` => `FanInConsumerGroupInfoClient`